Sources/PackageGraph/DependencyResolver.swift

```diff
@@ -675,7 +727,8 @@ public class DependencyResolver<
     func resolveSubtree(
         _ container: Container,
         subjectTo allConstraints: ConstraintSet,
-        excluding allExclusions: [Identifier: Set<Version>]
+        excluding allExclusions: [Identifier: Set<Version>],
+        containers: Set<Identifier>
```

Comment:
I'm not sure... it just feels unnecessary I didn't look through exactly what would be required to make that so.